### PR TITLE
{bio}[foss/2018b] DAS_Tool v1.1.1 w/ R 3.5.1 + Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#7712~~ (prodigal), ~~#7729~~ (BLAST+), ~~#7739~~ (Ruby), ~~#7740~~ (pullseq)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/92e03f8d0e2fef75edbb6fec1433df62 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d1c103b44455b04dbbb1c7203f61ff3f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/017746012f032c7427cacc992f87c199 for a full test report.

File: easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2018b-R-3.5.1-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    (name, version, {
+        'source_tmpl': 'DASTool_%(version)s.tar.gz',
+        'source_urls': ['https://github.com/cmks/DAS_Tool/raw/%(version)s/package/'],
+        'checksums': ['8d33997baaaec00d253b2d749cf1ace004ccdea2275b763d4d0f1c969916b72b'],
+        'modulename': 'DASTool',
+    }),
+]
+
+postinstallcmds = [
+    "cd %(installdir)s; unzip db.zip",
```
Comment (wpoely86) [inline easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2018b-R-3.5.1-Python-2.7.15.eb]: can you add something from this zip in the sanity check?

Comment (boegel) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2018b-R-3.5.1-Python-2.7.15.eb]: done in 4ed0e95

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2068.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5f51ee89ff3a741a1c59120d15e86323 for a full test report.

**Comment by akesandgren** [comment]
@boegel I'm having problems with this one failing sanity checks.
Error in library(DASTool) : there is no package called ‘DASTool’
Any ideas?

**Comment by boegel** [comment]
@akesandgren That basically indicates that the installation failed, any trace of an actual installation error higher up?

**Comment by akesandgren** [comment]
Gah, had a .Renviron file in $HOME...

**Comment by boegel** [comment]
@akesandgren Time to pick up on https://github.com/easybuilders/easybuild-easyblocks/issues/867?

**Comment by akesandgren** [comment]
yeah, probably...

**Pull request merged**